src/components/rewards-bar/index.test.tsx

```diff
@@ -92,6 +92,28 @@ describe('rewards-bar', () => {
     expect(wrapper.find(TooltipPopup).prop('open')).toBeFalse();
   });
 
+  it('displays rewards icon status when conditions are met', function () {
+    const wrapper = subject({
+      showNewRewards: true,
+      isMessengerFullScreen: true,
+    });
+
+    wrapper.setState({ isRewardsPopupOpen: false });
```

Comment:
It'd be preferable to perform the actual action that causes this state to get updated rather than have the test "just know" the internal state of the component. So, find the button and simulate the click event kinda thing.

Reply:
Nice suggestion thanks! I have updated :)